### ISSUE TITLE
[Feature Request] Option to disable all images and stickers in chat view

### ISSUE THREAD

**Issue by jorisw** [telegramdesktop/tdesktop#25690]
### Is your feature request related to a problem?

Many of the Telegram groups I use are full of users that think it's funny or interesting to post stickers, pictures, animated GIFs, etc, while I'm trying to use the group for serious discussion or offering ideas. All this noise makes it hard to follow the chat and degrades quality of content severely, and the Telegram user experience as a result

### Describe the solution you'd like

An option to not-preview stickers or pasted images in the chat view

### Describe alternatives you've considered

Ditching Telegram as a chat platform

### Additional context

TweetBot, a popular Mac Twitter client, has this option. Instead of pictures, you see the `pic.twitter.com` link that you can click to view.

**Comment by Aokromes** [comment]
duplicate.

**Comment by joriswitteman** [comment]
Could you link to it? I looked for it

**Comment by Aokromes** [comment]
https://github.com/telegramdesktop/tdesktop/issues/6999
